minecraft_servers/laidtofall/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "laidtofall",
+  "nice_name": "LaidToFall",
+  "direct_ip": "play.laidtofall.net",
+  "server_wildcards": [
+    "%.laidtofall.net"
+  ],
+  "social": {
+    "web": "https://laidtofall.net/",
+    "web_shop": "https://store.laidtofall.net/",
+    "instagram": "laidtofallmc",
+    "discord": "https://laidtofall.net/discord",
+    "youtube": "https://www.youtube.com/laidToFall69",
+    "teamspeak": "ts.labymod.net"
```

Comment:
I don't think that this is your teamspeak connection. Please update this to YOUR teamspeak IP.